[announcment] branding and angular 18 announcement (#2521)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-0-0-angular-18-upgrade-and-re-branding.md

---
date: 2024-10-15
title: Angular 18 upgrade and rebranding
product_area: Application enablement & solutions
change_type:
  - value: change-inv-3bw8e
    label: Announcement
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59795
version: 1021.0.0
---

> In a future version, we will update Angular to version 18 and update our base branding. This update brings improvements and new features but may also introduce breaking changes that could impact your existing implementations. Please verify your branding, especially the position of the logo was changed and needs to be checked if updating to this new version.

Reviewer: ```suggestion
In a future version, we will update Angular to version 18 and update our base branding. This update brings improvements and new features, but may also introduce breaking changes that could impact your existing implementations. When updating to this new version, please verify your branding, especially the logo position, as it has been modified.
```
